src/formio/components/checkbox.ts

```diff
@@ -0,0 +1,15 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type CheckboxInputSchema = InputComponentSchema<string, Validator, TranslatableKeys>;
```

Comment:
I'm not sure that `InputComponentSchema` is the right approach here because of the generic `defaultValue` type which makes the type wide enough to handle `multiple: false` and `multiple: true` - but the checkbox component does not allow `multiple`. I think you could use 

```ts
interface ... extends InputComponentSchema<...> {
  defaultValue: boolean;
}
```
to narrow it, but `infer`-ing based on `InputComponentSchema` will then probably return `boolean | boolean[]` which would be too wide. So personally, I'd opt to use lower-level building blocks for this component type.

Also note that the expected (default) value of a checkbox is `boolean`, so that generic type needs to be passed:

```suggestion
export type CheckboxInputSchema = InputComponentSchema<boolean, Validator, TranslatableKeys>;
```

Reply:
The following seems to work:

```ts
/**
 * @group Form.io components
 * @category Concrete types
 */
export interface CheckboxComponentSchema
  extends Omit<CheckboxInputSchema, 'hideLabel' | 'disabled'> {
  defaultValue: boolean;
  type: 'checkbox';
  multiple?: false,
}
```